doc-content/enterprise-only/openshift/hacep-server-maven-proc.adoc

```diff
@@ -0,0 +1,132 @@
++
+In the output of the command, find lines that list `supplemental-groups` and `uid-range`, similar to the following example:
++
+----
+openshift.io/sa.scc.supplemental-groups=1000160000/10000
+openshift.io/sa.scc.uid-range=1000160000/10000
+----
++
+. In the `springboot` directory, edit the `pom.xml` file to remove the following dependency:
++
+----
+<dependency>
+      <groupId>org.kie</groupId>
+      <artifactId>sample-hacep-project-kjar</artifactId>
+</dependency>
+----
++
+. In the `springboot` directory, edit the `Dockerfile` file. Find the following line:
++
+----
+RUN groupadd -r app -g <id_group> && useradd -u <id_user> -r -g app -m -d /app -s /sbin/nologin -c "App user" app && chmod 755 /app----
+----
+In this line, replace `<id_group>` with a value in the `supplemental-groups` interval and `<id_user>` with a value in the `uid-range` interval, as detected i nthe previous step, similar to the following example:
++
+----
+RUN groupadd -r app -g 1000160001 && useradd -u 1000160001 -r -g app -m -d /app -s /sbin/nologin -c "App user" app && chmod 755 /app
+----
++
+. In the `springboot` directory, enter the following commands to create the image for deployment and push it into the repository configured for your OpenShift environment:
++
+----
+oc new-build --binary --strategy=docker --name openshift-kie-springboot
+oc start-build openshift-kie-springboot --from-dir=. --follow
+----
++
+. Enter the following command to detect the name of the image that was built:
++
+----
+oc get is/openshift-kie-springboot -o template --template='{{range .status.tags}}{{range .items}}{{.dockerImageReference}}{{end}}{{end}}'
+----
++
+. Open the `kubernetes/deployment.yaml` file in a text editor.
+. Replace the existing image URL with the result of the previous command.
+. Replace `<id_user>` with the user ID that you entered in the Docker file in a previous step.
+. Remove all characters at the end of the line starting with the `@` symbol, then add `:latest` to  the line. For example: 
++
+----
+image: image-registry.openshift-image-registry.svc:5000/hacep/openshift-kie-springboot:latest
+----
++
+. Under the `containers:` line and the `env:` line, set environment variables as in the following example:
++
+----
+containers:
+  - env:
+    - name: UPDATABLEKJAR
+      value: "true"
+    - name: KJARGAV
+      value: <GroupID>:<ArtifactID>:<Version>
+    - name: MAVEN_LOCAL_REPO
+      value: /app/.m2/repository
+    - name: MAVEN_MIRROR_URL
+      value: http://<nexus_url>/repository/maven-releases/
+    - name: MAVEN_SETTINGS_XML
+      value: /app/.m2/settings.xml
+----
++
+In this example, replace the value of the `KJARGAV` variable with the group, artifact and version (GAV) of your KJAR service and the value of the `MAVEN_MIRROR_URL` variable with the URL to the Maven repository that contains your KJAR service.
++
+You can optionally set other variables. For a list of supported environment variables, see <<hacep-server-environment-ref>>.
```

Comment:
Remove "You can"

Optionally set other variables. For a list of supported environment variables, see <<hacep-server-environment-ref>>.